internal/instrumentation/metrics/business/README.md

```diff
@@ -0,0 +1,187 @@
+# Business Metrics Collectors
```

Comment:
We usually put docs under docs/
Is there anything in this file not already covered there?